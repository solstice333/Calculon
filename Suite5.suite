P a.out main.c header.h
T test1.in test1.out 1000
