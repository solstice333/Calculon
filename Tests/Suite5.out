Could not find required test file 'test1.in'
Nonzero exit status: 1
